common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -93,9 +93,9 @@ protected[core] case class BlackBoxExec(image: String) extends Exec(Exec.BLACKBO
     override val sentinelledLogs = false
 }
 
-protected[core] case class SequenceExec(code: String, components: Vector[String]) extends Exec(Exec.SEQUENCE) {
+protected[core] case class SequenceExec(code: String, components: Vector[FullyQualifiedEntityName]) extends Exec(Exec.SEQUENCE) {
     val image = Exec.imagename(Exec.NODEJS)
-    def size = components.map(_ sizeInBytes).reduce(_ + _)
+    def size = 0.bytes // not used for the hacky implementation and not used for the new implementation either
 }
 
 protected[core] object Exec
```

Comment:
after clarification with @ioana-blue, reverting this change in an upcoming commit.